### ISSUE TITLE
Incorrect option for mailx

### ISSUE THREAD

**Issue by klm1** [Matty9191/ssl-cert-check#34]
Using the **-a** and **-e** options on Debian 8 (jessie) as in

```
ssl-cert-check -f /etc/ssl-certs-list -x 14 -a -q -e user@domain.com
```

with ssl-cert-check produces this error:

```
/usr/bin/mailx: invalid option -- 'r'
usage: mailx [-dEIinv] [-a header] [-b bcc-addr] [-c cc-addr] [-s subject] to-addr ...
       mailx [-dEIiNnv] -f [name]
       mailx [-dEIiNnv] [-u user]
```

It appears that there is no **-r** option, at least for the mailx included with Debian 8.

**Comment by Matty9191** [comment]
Can you try version 4.1? I made some changes which will *hopefully* help.

**Comment by Matty9191** [comment]
This issue should be fixed in 4.9.
